app/incident/views.py

```diff
@@ -16,7 +16,7 @@
 # API Logic for Incident Category
 class IncidentCategoryView(View):
     def get(self, request):
-        lng = request.lng
+        lng = 'en-CA'
```

Comment:
While merging this got overwritten please check the changes present on repo. Make changes as needed it should be lng = request.lng everywhere.

Reply:
Make these changes please